packages/x-data-grid/src/models/gridBaseSlots.ts

```diff
@@ -1,5 +1,19 @@
 type Ref<T = HTMLElement> = React.RefCallback<T | null> | React.RefObject<T | null> | null;
 
+type CommonProps = {
+  className?: string;
+  style?: React.CSSProperties;
+  onClick?: React.MouseEventHandler;
+  onDoubleClick?: React.MouseEventHandler;
+  onPointerUp?: React.MouseEventHandler;
+  onPointerDown?: React.MouseEventHandler;
+  onKeyDown?: React.KeyboardEventHandler;
+  onKeyUp?: React.KeyboardEventHandler;
+  onFocus?: React.FocusEventHandler;
+  onBlur?: React.FocusEventHandler;
+  [k: `aria-${string}`]: any;
+};
```

Comment:
If a prop isn't present here, we can't use it in the core datagrid. So if we'd need e.g. `onTouchStart` on a base component later on, we couldn't use it. I'm wondering if I should add all event handlers here?

Reply:
I see that you used `React.DOMAttributes` – makes sense to me